content/en/network_performance_monitoring/installation.md

```diff
@@ -27,6 +27,10 @@ Supported platforms include:
 
 There is an exemption to the 4.4.0+ kernel requirement for [CentOS/RHEL 7.6+][2]. The [DNS Resolution][3] feature is not supported on CentOS/RHEL 7.6.
 
+Network Performance Monitoring is compatible with Cilium installations, provided the following requirements are met:
```

Comment:
```suggestion
Network Performance Monitoring is compatible with **Cilium** installations, provided the following requirements are met:
```
Motivation:
The number of requirements + exceptions is starting to get lengthy + hard to parse - I would bold "Cilium", "provisioning systems", and "platform" above for readability.